corehq/sql_db/util.py

```diff
@@ -365,7 +365,7 @@ def get_standbys_with_acceptible_delay():
 
     return {
         db_ for db_, delay in delays_by_db.items()
-        if 0 <= delay <= acceptable_delays_by_db[db_]
+        if 0 <= delay or -1 <= acceptable_delays_by_db[db_]
```

Comment:
This would need brackets in order to work. It also has a problem when `delay = 0` since `0 or -1 == -1`.

Reply:
👍 Will update